VirtoCommerce.Domain/Catalog/Model/Category.cs

```diff
@@ -1,10 +1,11 @@
+using System;
 using System.Collections.Generic;
 using VirtoCommerce.Domain.Commerce.Model;
 using VirtoCommerce.Platform.Core.Common;
 
 namespace VirtoCommerce.Domain.Catalog.Model
 {
-    public class Category : AuditableEntity, ILinkSupport, ISeoSupport, IHasOutlines, IHasImages, IHasProperties
+    public class Category : AuditableEntity, ILinkSupport, ISeoSupport, IHasOutlines, IHasImages, IHasProperties, ICloneable
```

Comment:
Please make required objects and related ones ICloneable the same way as in this commit (it is not merged in v3 yet):
https://github.com/VirtoCommerce/vc-platform-core/pull/434/commits/7c33eb647dd6cd000ac4c92bdc0212cfe7d71e46